vector-im/element-android
# Space Switching: Accordion-Style Bottom Sheet

## Type of change


## Content

Changes the space sheet style back to accordion style as it is in the old app layout

Other not-so-obvious changes:
- Long clicking on a space now opens space settings for that space instead of drag-and-drop repositioning

## Motivation and context

Closes https://github.com/vector-im/element-android/issues/6815

## Screenshots / GIFs

| Light | Dark | Black |
|-|-|-|
| <img width="200" alt="image" src="https://user-images.githubusercontent.com/20701752/185922487-e833aaee-77f3-46e7-9870-5a093ab901f7.png"> | <img width="200" alt="image" src="https://user-images.githubusercontent.com/20701752/185922616-a8f49a4f-1339-4cc6-a607-c4907078d84c.png"> | <img width="200" alt="image" src="https://user-images.githubusercontent.com/20701752/185922759-a70c1817-1b7f-45ad-b8ab-34193ea5cd2e.png"> |

## Tests

- Open the space sheet
- See that spaces that have subspaces can be expanded
- See that this holds true for subspaces too
- See that selected space / subspace is highlighted grey
- See that long clicking space / subspace opens space settings for that space

## Tested devices

- OS version(s): Android 12

## Checklist
